src/utils/admin-utils.ts

```diff
@@ -0,0 +1,26 @@
+import { FeatureCollection } from 'geojson';
+
+import {
+  getBoundaryLayerSingleton,
+  getDisplayBoundaryLayers,
+} from '../config/utils';
+
+export function getAdminLayerURL(adminLevel = 1): string {
+  const layers = getDisplayBoundaryLayers();
+  const adminLayer = layers.find(
+    layer => layer.adminLevelNames.length === adminLevel,
+  );
+  return adminLayer ? adminLayer.path : layers[0].path;
+}
+
+export function getAdminNameProperty(adminLevel: number = 1): string {
```

Comment:
This as well looks like it can use the new AdminLevel type

Reply:
fixed with https://github.com/WFP-VAM/prism-frontend/pull/431/commits/9ef1104d3392e80ea63ffc36ca87b1bbe25304a5